packages/api/tests/models/test-collections-model.js

```diff
@@ -100,13 +87,21 @@ test('Collection.delete() throws an exception if the collection has associated r
 test('Collection.delete() deletes a collection', async (t) => {
   const name = randomString();
   const version = randomString();
+  const item = fakeCollectionFactory({ name, version });
+  const { dataType } = item;
+  const collectionConfigStore = collectionsModel.collectionConfigStore;
+  const collectionId = constructCollectionId(dataType, version);
 
-  await manager.create(fakeCollectionFactory({ name, version }));
-
-  t.true(await manager.exists({ name, version }));
+  await collectionsModel.create(item);
+  t.true(await collectionsModel.exists(name, version));
+  t.truthy(await collectionConfigStore.get(dataType, version));
 
-  const collectionsModel = new Collection();
   await collectionsModel.delete({ name, version });
-
-  t.false(await manager.exists({ name, version }));
+  t.false(await collectionsModel.exists(name, version));
+  // If the collection was successfully deleted from the config store, we
+  // expect attempting to get it from the config store to throw an exception.
+  await t.throwsAsync(
+    async () => collectionConfigStore.get(dataType, version),
+    { message: new RegExp(`${collectionId}`) }
```

Comment:
Genuine question: is it better to test the error message or the error class? I guess the error class doesn't give us any guarantee that it's referring to the collection we care about?